src/main/java/seedu/agendum/model/task/ChildRecurringTask.java

```diff
@@ -0,0 +1,81 @@
+package seedu.agendum.model.task;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+public class ChildRecurringTask extends RecurringTask {
```

Comment:
I believe in the abstraction occurrence pattern, the occurence should not extend the abstraction